app/(public-pages)/about/page.tsx

```diff
@@ -1,9 +1,17 @@
 import React from 'react'
 import { Metadata } from 'next'
 import { AboutPageSection } from '@/app/(public-pages)/_components/about-page/AboutPageSection'
+import OurTeam from '../_components/about-page/OurTeam';
```

Comment:
please try to use Absolute imports just like the type used above. it makes the codebase cleaner and ensures that we all use the same patterns